data/mp/stats/weapons.json

```diff
@@ -3166,6 +3166,51 @@
 		"weaponWav": "rocket.ogg",
 		"weight": 400
 	},
+	"Missile-VTOL-LtAAM": {
+		"buildPoints": 800,
+		"buildPower": 250,
+		"damage": 491,
+		"designable": 1,
+		"effectSize": 100,
+		"explosionWav": "smlexpl.ogg",
+		"faceInFlight": 1,
+		"facePlayer": 1,
+		"firePause": 5,
+		"flags": "AirOnly",
+		"flightGfx": "FXMflare.PIE",
+		"flightSpeed": 2250,
+		"hitGfx": "FXMExp.PIE",
+		"hitpoints": 50,
+		"id": "Missile-LtSAM",
```

Comment:
The id should be "Missile-VTOL-LtAAM" as well.

Reply:
Oops.